content-src/asrouter/components/Button/_Button.scss

```diff
@@ -7,7 +7,7 @@
   font-family: inherit;
   padding: 8px 15px;
   margin-inline-start: 12px;
-  color: inherit;
+  color: $grey-90-80;
```

Comment:
This doesn't work for dark theme, is there a variable maybe you could use?

Reply:
Ahh good point.

Reply:
looks like for default/secondary buttons, what you want is:

regular theme:
background: grey-10 
border: grey-30
text color: inherit

dark:
background: grey-60
border: grey-10-10

Maybe worth adding those to theme.scss?